dataactvalidator/scripts/get_city_file.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import os
+import requests
+import logging
+import re
+import boto3
+import urllib.request
+from bs4 import BeautifulSoup
+import zipfile
+import datetime
+
+from dataactcore.config import CONFIG_BROKER
+from dataactcore.logging import configure_logging
+
+from dataactvalidator.health_check import create_app
+
+logger = logging.getLogger(__name__)
+BASE_URL = 'https://geonames.usgs.gov'
+
+
+def get_download_url():
```

Comment:
since `get_download_url` is only called once, I don't think this needs to be its own function. The script itself is designed to do one thing... download the file.

Returning the file name also isn't necessary, as it's in the first returned value.

Reply:
likewise, the download function can be in-line with the flow of the script. it's a short script and simple enough to follow.

Reply:
I think this comes down to personal preference. In Broker, we rarely have a script that's JUST the main function and I'd argue that these are reasonable logical chunks